src/lib/ui/thermostat/ClimateSettingScheduleCard.tsx

```diff
@@ -0,0 +1,99 @@
+import type { ClimateSettingSchedule } from 'seamapi'
+
+import { formatDateAndTime } from 'lib/dates.js'
+import { ClimateSettingScheduleIcon } from 'lib/icons/ClimateSettingSchedule.js'
+import { useClimateSettingSchedule } from 'lib/seam/thermostats/climate-setting-schedules/use-climate-setting-schedule.js'
+import { ClimateSettingDevice } from 'lib/ui/thermostat/ClimateSettingScheduleDevice.js'
+import { ClimateSettingStatus } from 'lib/ui/thermostat/ClimateSettingStatus.js'
+
+import { DotDivider } from '../layout/DotDivider.js'
+
+interface ClimateSettingScheduleCardProps {
+  climateSettingScheduleId: string
+}
+
+export function ClimateSettingScheduleCard({
+  climateSettingScheduleId,
+}: ClimateSettingScheduleCardProps): JSX.Element {
+  return (
+    <div className='seam-climate-setting-schedule-card'>
+      <Content climateSettingScheduleId={climateSettingScheduleId} />
+    </div>
+  )
+}
+
+function Content(props: {
+  climateSettingScheduleId: string
+}): JSX.Element | null {
+  const { climateSettingScheduleId } = props
+
+  const { climateSettingSchedule } = useClimateSettingSchedule(
+    climateSettingScheduleId
+  )
+
+  if (climateSettingSchedule == null) {
+    return null
+  }
+
+  const name = climateSettingSchedule.name ?? t.fallbackName
+
+  return (
+    <div className='seam-climate-setting-schedule-content'>
+      <div className='seam-climate-setting-schedule-summary-container'>
+        <div className='seam-climate-setting-schedule-icon-block'>
+          <ClimateSettingScheduleIcon />
+        </div>
+        <div className='seam-climate-setting-schedule-summary'>
+          <h5 className='seam-climate-setting-schedule-heading'>{name}</h5>
+          <div className='seam-climate-setting-schedule-subheading'>
+            <ClimateSettingStatus climateSetting={climateSettingSchedule} />
+            <DotDivider />
+            <ClimateSettingScheduleTiming
+              climateSettingSchedule={climateSettingSchedule}
+            />
+          </div>
+        </div>
+      </div>
+      <ClimateSettingDevice deviceId={climateSettingSchedule.device_id} />
+    </div>
+  )
+}
+
+function ClimateSettingScheduleTiming(props: {
+  climateSettingSchedule: ClimateSettingSchedule
+}): JSX.Element | null {
+  const { climateSettingSchedule } = props
+
+  const startTime = new Date(
+    climateSettingSchedule.schedule_starts_at
+  ).getTime()
+
+  const endTime = new Date(climateSettingSchedule.schedule_ends_at).getTime()
+
+  if (Date.now() < startTime)
+    return (
+      <span>
+        {`${t.starts} ${formatDateAndTime(
+          climateSettingSchedule.schedule_starts_at
+        )}`}
+      </span>
+    )
+
+  if (startTime <= Date.now() && Date.now() <= endTime)
+    return (
+      <span>
+        {`${t.ends} ${formatDateAndTime(
+          climateSettingSchedule.schedule_starts_at
+        )}`}
```

Comment:
JSX is the template 😉 
```suggestion
        {t.ends} {formatDateAndTime(climateSettingSchedule.schedule_starts_at)}
```

Reply:
https://github.com/seamapi/react/pull/378/commits/dd76de64d418e71e13b2152e1c6e37b43a7c91c5#diff-be6143e4765e91974dd1ffab7a4a0c557a97a45e94ff32ca6904bbee0f72c662R80-R81

Reply:
I often use string templating because there are instances where the space gets dropped. That could be on me not understanding when jsx puts a space in there or not.

Reply:
Ah, yea good point. The way this is currently handles is via `{' '}`, see https://github.com/seamapi/react/blob/1c2b2d40197a2c8dce469fad9f666f80fa1335d1/src/lib/seam/components/DeviceTable/DeviceRow.tsx#L60 and

Reply:
Let's pick one way to manage this issue, that way if we find a better way we can find/replace for these cases and catch them all.

Reply:
Wild idea: `t.space` ?

Reply:
Woa, too far man >_< But for real, in this case the space is layout not i18n related.